### ISSUE TITLE
Display format and get other format

### ISSUE THREAD

**Issue by nezamy-wow-ws** [kenhyuwa/litepie-datepicker#11]
Hello,
How I can display date format in input and get another format as a value

**Comment by kenhyuwa** [comment]
you can format manually before send value to server.

**Comment by nezamy-wow-ws** [comment]
Yes I can do that but I prefer to get the value from the component
What I mean, I want to display the date format by example showing month name  in Arabic and get other format in value
```
<LitepieDatepicker
      i18n="ar"
      :formatter="displayFormatter"
      :value-formatter="valueFormatter"
      :disable-date="disableDate"
      v-model="displayDate"
      v-model:value="valueDate"
/>
```

**Comment by kenhyuwa** [comment]
Okay, 
I'm not sure if everyone needs this feature or not.

**Comment by alesf** [comment]
It's surely missed.
I want to display a nicely formatted date in input but send ISO formatted dates to API.

**Comment by kenhyuwa** [comment]
you can format before send to API.
but I not sure for adding this feature.

**Comment by lupanom** [comment]
Hi! These feature could be useful for me too! Thanks

**Comment by sentiasa** [comment]
That'd be great for me too.

**Comment by afunnydev** [comment]
@kenhyuwa Any ETA for that? Need any help? Not sure how I would approach this but I could probably help.

**Comment by muarachmann** [comment]
@kenhyuwa I think this feature would be great. Any ETA?

**Comment by manogi** [comment]
Commenting to politely add my voice to support this request. It would be great to be able to format both separately (the shown date and the date sent to the API).

**Comment by bpolaszek** [comment]
Voting up too, because it's not only a matter of normalizing before sending to server, but also denormalizing server data to populate daterange picker values.

People in Europe are used to the `DD/MM/YYYY` format, but we store dates as `YYYY-MM-DD`. So when I initialize a date value with `[YYYY-MM-DD, YYYY-MM-DD]` format, it yells because the formatter wants an `DD/MM/YYYY` format to pretty-print it to the user.
